rust/rope/src/delta.rs

```diff
@@ -205,10 +211,10 @@ impl<N: NodeInfo> Delta<N> {
         Delta { els: els, base_len: l }
     }
 
-    /// Return a subset that inverts the insert-only delta:
+    /// Return a Subset containing the inserted ranges of an insert-only delta.
```

Comment:
Something that isn't clear to me: is "insert-only" a guarantee made anywhere? Is the caller expected to ensure that a given delta is an "insert-only delta"?